### PR TITLE
[TST] Notebook autobuilding in CI system

### PR DESCRIPTION
**Pull request by ericmjl**
This PR adds a step to the CI system to check that the notebooks run correctly.

I figured it's probably better to just execute all notebooks and not introduce a fork in the build pipeline, since review takes time, and all of the dependencies are needed anyways.

### CODE REVIEW

**Comment by ericmjl** [comment]
Help @fonnesbeck and/or @ColCarroll! Any idea what's going on with `pm.utils.get_data()` here?

File: nbconfig.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+c = get_config()
+c.NbConvertApp.notebooks = [
```
[the remark this answers is not in the document]
Comment (ericmjl) [reply nbconfig.py]: Hmmm, I hear where you're coming from, but if we don't execute the long-running ones, then we should be ok, maybe? It's with this consideration that I thought of not using globbing but instead explicitly marking out which ones to execute. (We get to hit two birds with one stone then: integration test + R-style vignettes.)

[the remark this answers is not in the document]
Comment (eigenfoo) [reply nbconfig.py]: Oh hmmm I didn't really think of it that way! I guess I like this setup, as long as we agree on which notebooks to run each time.

Perhaps a docstring here would help clarify?

[the remark this answers is not in the document]
Comment (ericmjl) [reply nbconfig.py]: Cool beans - coming up!

**Comment by ericmjl** [comment]
@twiecki and @junpenglao, I’m leaving out the Radon notebook for now, as I wasn’t feeling confident in being able to modify it correctly while preserving the pedagogical goals.

Let’s add it back to the CI system in a later PR, how’s that?

**Comment by twiecki** [comment]
Sounds good!

**Comment by ericmjl** [comment]
Woohoo! Passed - it never feels right for me to hit the green button on my own PRs; anybody else want to do it?

**Comment by twiecki** [comment]
👍

**Pull request merged**